src/bootstrap.php

```diff
@@ -588,59 +550,38 @@ function (Request $request, string $number, int $version) use ($app) {
             ];
         }
 
-        /** @var Accept $type */
-        $type = $app['negotiator']->getBest($request->headers->get('Accept'), $accepts);
-
-        $version = (int) $type->getParameter('version');
-        $type = $type->getType();
+        $app['content_negotiator.accept']->negotiate($request, $accepts);
+        $type = $request->attributes->get(ContentNegotiationProvider::ATTRIBUTE_ACCEPT);
```

Comment:
This is where it's a little tricky, the route produces either a VoR or a PoA type depending on the version.